.github/workflows/wda-package.yml

```diff
@@ -45,10 +48,26 @@ jobs:
             CODE_SIGNING_ALLOWED=NO ARCHS=arm64
       - name: Creating a zip of WebDriverAgentRunner-Runner.app for tvOS
         run: |
-          pushd appium_wda_tvos/Build/Products/Debug-appletvos
+          pushd $PKG_PATH_TVOS/Build/Products/Debug-appletvos
           zip -r $ZIP_PKG_NAME_TVOS WebDriverAgentRunner_tvOS-Runner.app
           popd
           mv $PKG_PATH_TVOS/Build/Products/Debug-appletvos/$ZIP_PKG_NAME_TVOS ./
+
+      - name: Build iOS for simulator
+        run: |
+          xcodebuild clean build-for-testing \
+            -project WebDriverAgent.xcodeproj \
+            -derivedDataPath $PKG_PATH_IOS_SIM \
+            -scheme WebDriverAgentRunner \
+            -destination "platform=iOS Simulator,name=iPhone 14 Pro,OS=16.2" \
```

Comment:
Should we hardcode the platform version?

Reply:
Maybe not. Then, the version will be the Xcode's default one. Currently this is to clarify if the OS version affects running simulator version.

e.g. build for OS 16.2 can work with 15.5

Lower one worked, but not sure higher one can work right now.